cli/src/code_coverage/prvsnr-coverage.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+import argparse
+import subprocess
+import sys
+import os.path
+import shlex
```

Comment:
It's a good practice to arrange the imports alphabetically.

Reply:
Addressed